.generated-sources/emily/build.rs

```diff
@@ -0,0 +1,47 @@
+use std::process::Command;
+use std::env;
+
+fn main() {
+
+    // Ensure that we rerun if the API changes.
+    println!("cargo:rerun-if-changed=../../emily/api-definition/models");
+    println!("cargo:rerun-if-changed=build.rs");
```

Comment:
Is the goal of this line to prevent this build script from being rerun all the time? I guess to cut down on the overhead of the `npm` commands?

Reply:
yep